src/screens/Dashboard/CompactTickets.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {useState} from 'react';
+import {StyleSheet} from '@atb/theme';
+import * as Sections from '@atb/components/sections';
+import {
+  ShortTicketInfoView,
+  mapToUserProfilesWithCount,
+} from '../Ticketing/Ticket/TicketInfo';
+import {isPreactivatedTicket, isInspectableTicket} from '@atb/tickets';
+import {getValidityStatus} from '@atb/screens/Ticketing/Ticket/utils';
+import useInterval from '@atb/utils/use-interval';
+import {
+  useHasEnabledMobileToken,
+  useMobileTokenContextState,
+} from '@atb/mobile-token/MobileTokenContext';
+import {findReferenceDataById} from '@atb/reference-data/utils';
+import {useFirestoreConfiguration} from '@atb/configuration/FirestoreConfigurationContext';
+import {
+  filterActiveOrCanBeUsedFareContracts,
+  isValidRightNowFareContract,
+  useTicketState,
+} from '@atb/tickets';
+import ThemeText from '@atb/components/text';
+import {TicketsTexts, useTranslation} from '@atb/translations';
+
+type Props = {
+  onPressDetails?: (orderId: string) => void;
+};
+
+const CompactTickets: React.FC<Props> = ({onPressDetails}) => {
+  const itemStyle = useStyles();
+
+  // TODO: Verify if this is the correct way to update time-to-time the expiration time of the tickets.
+  const [now, setNow] = useState<number>(Date.now());
+  useInterval(() => setNow(Date.now()), 1000);
```

Comment:
I noticed that the ticket is not updating to present an updated text for validity, I made this solution based on current solutions in the code. 

NOTE: The details view is not being updated when the time changes. (Maybe another task)